src/libraries/System.Private.DataContractSerialization/src/System/Runtime/Serialization/CollectionDataContract.cs

```diff
@@ -1038,7 +1087,14 @@ internal static bool TryCreateGetOnlyCollectionDataContract(Type type, [NotNullW
             }
         }
 
-        internal static MethodInfo? GetTargetMethodWithName(string name, Type type, Type interfaceType)
+        [UnconditionalSuppressMessage("ReflectionAnalysis", "IL2075:MakeGenericMethod",
```

Comment:
`MakeGenericMethod`?

Reply:
TBH I don't really recall why I added these here but they look like a copy-paste error. I have removed this suppression as well as the other suppression I introduced which also wasn't required. I've checked the remaining Suppressions that I added and they all seem correct now.